### PR TITLE
Ns/icon jenkins test

### PR DESCRIPTION
**Pull request by NadyaShulga**
https://trello.com/c/rkil4C3t/51-tc0202001-verify-icon-and-logo-on-the-first-page

### CODE REVIEW

**Comment by RuslanG13** [comment]
Hello) 
We should follow the rule "one test-case card = one it block". You can combine two its into one it and use them in one cy.js file

File: cypress/e2e/iconJenkinsTest.cy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+///<reference types ="cypress"/>
+
+describe ('iconJenkins', () =>{
+
+    it('Verify icon and logo on the first page', function (){
```
Comment (RuslanG13) [inline cypress/e2e/iconJenkinsTest.cy.js:5]: It titles should contain the title of the test case card on the Trello board. f.e. TC_02.02.001 | Verify icon and logo on the first page

File: cypress/e2e/iconJenkinsTest.cy.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+///<reference types ="cypress"/>
+
+describe ('iconJenkins', () =>{
```
Comment (RuslanG13) [inline cypress/e2e/iconJenkinsTest.cy.js:3]: Title of describe and name of the iconJenkinsTest does not match

File: cypress/e2e/iconJenkinsTest.cy.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+    it('Verify icon and logo on the first page', function (){
+cy.get('#jenkins-head-icon').click().should('be.visible');
+cy.get('#jenkins-name-icon').click().should('be.visible');
+
+    });
```
Comment (RuslanG13) [inline cypress/e2e/iconJenkinsTest.cy.js:9]: You can use the Prettier extension in VS for formatting your code with spaces to beautify your code.
And you can delete this extra empty rows in your code if you want

**Comment by NadyaShulga** [comment]
@RuslanG13 Hi, Ruslan. Thank you for your respond. What is better to do now? To create a new PR with two funcrions in one cy.file?

File: cypress/e2e/iconJenkinsTest.cy.js (hunk 3 above)
Comment (NadyaShulga) [reply cypress/e2e/iconJenkinsTest.cy.js:9]: Hi, Ruslan. Thank you for your respond. What is better to do now? To create a new PR with two funcrions in one cy.file?

File: cypress/e2e/iconJenkinsTest.cy.js (hunk 2 above)
Comment (NadyaShulga) [reply cypress/e2e/iconJenkinsTest.cy.js:3]: Agreed

File: cypress/e2e/iconJenkinsTest.cy.js (hunk 1 above)
Comment (NadyaShulga) [reply cypress/e2e/iconJenkinsTest.cy.js:5]: Maria Moroz showed us Title without info about number of TC

**Pull request closed without merging**